### ISSUE TITLE
Client ID

### ISSUE THREAD

**Issue by Blazayy** [MJC17/Nike-Stock-Moniter#1]
Hi ! 

Could you help on how to find the client ID for UK ? 

Thanks 👍

**Comment by MJC17** [comment]
Yeah, no problem but why do you need it. just wondering... like what are the uses of the client ID

**Comment by Blazayy** [comment]
Like on the app page when you enter a SKU then location it ask for client ID to make a search, does the app can show stocks of uncomming pairs ?

**Comment by MJC17** [comment]
Oh I now get what you mean. Yes, when I'm done the app you will be able to the sizes in stock and hopefully it will also show the stock of upcoming pairs. I'm honestly having a hard time finding that data within the JSON but I'm working on it. I'm probably go to take out the client ID section and replace it with an nike login, it real isn't need. So don't worry about that.

do you know any python or Javascript/JSON? ... I need some help with some parts of my code
